frontend/src/components/Brokers/Broker/Configs/TableComponents/InputCell/styled.ts

```diff
@@ -1,22 +1,23 @@
 import styled from 'styled-components';
 
-export const ValueWrapper = styled.div<{ $isDynamic?: boolean }>`
+export const ValueWrapper = styled.div`
   display: flex;
   justify-content: space-between;
-  font-weight: ${({ $isDynamic }) => ($isDynamic ? 600 : 400)};
```

Comment:
could you explain why we're moving these properties to the value rather than its' wrapper?

Reply:
The previous code made "Property is read-only" bold only for dynamic values in read-only mode, which seems a little weird at that time for me. I expected only the value itself to be bold.
![image](https://github.com/user-attachments/assets/ba74f1f2-d43a-4298-9637-eeb0acd211de)


BTW, I do not insist in keeping this change.